os/arch/arm/src/amebad/Kconfig

```diff
@@ -67,4 +67,16 @@ config AMEBAD_TRUSTZONE
 	default n
 	select ARMV8M_TRUSTZONE
 
+config AMEBAD_RDP
+	bool "Enable Amebad RDP"
+	default n
+	select AMEBAD_RDP_ENABLE
+	depends on AMEBAD_TRUSTZONE
+
+config AMEBAD_RDP_KEY
```

Comment:
Is it okay not to set the default value?

Reply:
Are you talking about the RDP key?
Default key in RTL8721CSM is [2473040ab47c48655a15aa431c4bbb8a], so if user wanto modify they have delete or backspace all the way, is that okey?

Reply:
Yes. I think we'd better show the default key in here.

Reply:
Okay, i will add the default key here. Thanks